inspirehep/config.py

```diff
@@ -59,7 +59,7 @@ def _(x):
 
 # Deal with inconcistency :puke:
 PID_TYPES_TO_ENDPOINTS = {"lit": "literature", "aut": "authors"}
-PID_TYPES_TO_SCHEMA = {"hep": "lit", "authors": "aut"}
+PID_TYPES_TO_SCHEMA = {"hep": "lit", "authors": "aut", "jobs": "job"}
```

Comment:
```suggestion
PID_TYPES_TO_SCHEMA = {"hep": "lit", "authors": "aut", "job": "jobs"}
```

Reply:
Actually here the field should be renamed to SCHEMA_TO_PID_TYPES and it should remain "jobs": "job"

Reply:
Indeed, it's only used here https://github.com/inspirehep/inspirehep/blob/2588356b81ffa8a92ebb8a65d4d54b15dcc42e9b/inspirehep/pidstore/api/base.py#L70 could you please also refactor this part? it's just a small renaming.

Reply:
yes, of course